### ISSUE TITLE
Create log file for CheckM test

### ISSUE THREAD

**Issue by vinisalazar** [Ecogenomics/CheckM#179]
The command `checkm test` output a console log stating if the tests passed or not. I suggest outputting this information to a `test.log` file in the test results directory. I am quite sure this can be done with the `logger` module.

**Comment by vinisalazar** [comment]
I tried redirecting the test command to a test file and it gave me a rudimentary solution.
On the Terminal, I ran:
`$ checkm test test_4 > test_4.txt`

Contents of `test_4.txt`:
```
[Step 1]: Verifying tree command.

  [Passed]


[Step 2]: Verifying tree_qa command.

  [Passed]


[Step 3]: Verifying lineage_set command.

  [Passed]


[Step 4]: Verifying analyze command.

  [Passed]


[Step 5]: Verifying qa command.

  [Passed]
```

**Comment by dparks1134** [comment]
This will be implemented in v1.0.15 which I plan to release later today.
